sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -897,6 +897,13 @@ object SQLConf {
       .booleanConf
       .createWithDefault(true)
 
+  val PARQUET_VECTORIZED_READER_NESTED_COLUMN_ENABLED =
+    buildConf("spark.sql.parquet.enableNestedColumnVectorizedReader")
+      .doc("Enables vectorized Parquet decoding for nested columns (e.g., struct, list, map).")
+      .version("3.3.0")
+      .booleanConf
+      .createWithDefault(false)
```

Comment:
As we don't push this into 3.2 and there is still a while before 3.3. We already have a config for vectorized reader. We may not need a separate config.

Reply:
oh okay I'm fine with that too - I basically followed ORC to add a similar config here.

Reply:
I'm fine to have this. For 3.3.0, shall we start with `true`? We can disable it later during `branch-3.3` preparation.

Reply:
If we want to have it, default as true sounds better as it will be tested by default. We still have time to disable it before it is actually released.

Reply:
okay, I'll keep it and set it to true by default, thanks!